### PR TITLE
Add path component to file:FileInfo

### PR DESCRIPTION
**Pull request by praveennadarajah**
## Purpose
This PR will add path component to file:FileInfo.

Fixes: https://github.com/ballerina-platform/ballerina-lang/issues/19149

### CODE REVIEW

File: composer/package-lock.json [hunk 1]
```diff
@@ -11408,8 +11408,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```
Comment (chamil321) [inline composer/package-lock.json]: Shall we revert this change done to the package-lock.json file? I think you have mistakenly committed it too

Comment (praveennadarajah) [reply composer/package-lock.json]: Yeas. I have reverted it. Committed mistakenly.

File: stdlib/file/src/main/java/org/ballerinalang/stdlib/file/utils/FileUtils.java [hunk 2]
```diff
@@ -89,7 +89,7 @@ public static ObjectValue getFileInfo(File inputFile) throws IOException {
         lastModifiedInstance = createTimeRecord(getTimeZoneRecord(), getTimeRecord(),
                 lastModified.toMillis(), zonedDateTime.getZone().toString());
         return BallerinaValues.createObjectValue(FILE_PACKAGE_ID, FILE_INFO_TYPE, inputFile.getName(),
-                inputFile.length(), lastModifiedInstance, inputFile.isDirectory());
+                inputFile.length(), lastModifiedInstance, inputFile.isDirectory(), inputFile.getPath());
```
Comment (daneshk) [inline stdlib/file/src/main/java/org/ballerinalang/stdlib/file/utils/FileUtils.java]:
```suggestion
                inputFile.length(), lastModifiedInstance, inputFile.isDirectory(), inputFile.getAbsolutePath());
```
